README.rst

```diff
@@ -14,8 +14,7 @@ Welcome to Kiwi
     :alt: Documentation Status
 
 Kiwi is an efficient C++ implementation of the Cassowary constraint solving
-algorithm. Kiwi is an implementation of the algorithm based on the seminal
-Cassowary paper. It is *not* a refactoring of the original C++ solver. Kiwi
+algorithm. Kiwi is an implementation of the algorithm based on the `seminal Cassowary paper <https://constraints.cs.washington.edu/solvers/cassowary-tochi.pdf>`_. It is *not* a refactoring of the original C++ solver. Kiwi
```

Comment:
I dislike having such long lines. It makes editing harder I find. Any chance you could manage to fold it ?

Reply:
Done